e2e-tests/specs/sample.js

```diff
@@ -0,0 +1,10 @@
+const { expect, $ } = require('@wdio/globals');
+
+describe('Connect test', () => {
+  it('should call app successfully', async () => {
+    const selector = driver.isAndroid
+      ? await $('android=new UiSelector().className("android.widget.FrameLayout")')
```

Comment:
Can you explain to me what `$` does in this instance? Is this how we select elements? (sort of like jQuery?)

Reply:
Yes, $ command is a short and handy way in order to fetch a single element on the page.
You can check more information [here](https://webdriver.io/docs/api/element/$/)